mbs_results/growth_rate_output.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import pandas as pd
+
+
+def get_growth_rate_data(filepath):
```

Comment:
type hint for filepath in function input?
`filepath: str` 

Also could update the return Dtype to just pd.DataFrame if you were also changing the above